gframe/menu_handler.h

```diff
@@ -281,6 +281,7 @@ enum GUI {
 	CHECKBOX_ENABLE_MUSIC,
 	CHECKBOX_ENABLE_SOUND,
 	CHECKBOX_IGNORE_DECK_CONTENTS,
+	CHECKBOX_ADD_CARD_NAME_TO_DECK_LIST,
```

Comment:
The area here at the end is for synchronized elements, you should put the entry above, it can be put below `CHECKBOX_CONFIRM_DECK_CLEAR`